### ISSUE TITLE
voltocli fails with "TypeError: git.branch is not a function"

### ISSUE THREAD

**Issue by tisto** [nzambello/voltocli#9]
````
➜  npx voltocli
npx: Installierte 334 in 25.165s
✔ Pick an action › create
✔ What's your new addon name? … volto-rainbow-widget
✔ Insert git repository URL … git@github.com:collective/volto-rainbow-widget.git
✔ Insert a description … Volto addon to build beautiful things
✔ Insert author name … collective
✔ Preparing the environment
✔ Cloning template

TypeError: git.branch is not a function
````

**Comment by nzambello** [comment]
That's for master vs main default branch
I'll check if I can retrieve the branch instead of assuming it, because it's developers local configuration

**Comment by nzambello** [comment]
@tisto fixed and released in [v2.1.3](https://github.com/nzambello/voltocli/releases/tag/v2.1.3)
